marcdown_project/marcdown/urls.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import path, include
+from django.contrib import admin
+
+from . import views
+from . import api_views
+
+from rest_framework import routers
+from rest_framework.urlpatterns import format_suffix_patterns
+
+app_name = "marcdown"
+
+router = routers.DefaultRouter()
+router.register(r'user', api_views.UserViewSet, basename='user')
```

Comment:
Is basename required if the first argument is always the same?

Reply:
The ViewSet used is not a ModelViewSet, and thus the basename cannot be inferred.